### PR TITLE
[All] (SSO) Update instructions for new app registration portal

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[1f72616](https://github.com/OfficeDev/office-js-docs-pr/pull/886/commits/1f726162f20a3e8e957b4c8ba0adbeee31ce1657)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/register-sso-add-in-aad-v2.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/develop/register-sso-add-in-aad-v2.md) | :white_check_mark:Succeeded | |
[docs/develop/sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/develop/sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/includes/grant-admin-consent-to-an-add-in-include.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/includes/grant-admin-consent-to-an-add-in-include.md) | :white_check_mark:Succeeded | |
[docs/includes/register-sso-add-in-aad-v2-include.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/includes/register-sso-add-in-aad-v2-include.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C10%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904102036318262-886%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=M5NRUFMtrpzMUmxEvCSr%2FPdZKUfFSBPrAB48uGvJcR4%3D&st=2019-04-10T20%3A32%3A50Z&se=2019-05-11T20%3A37%3A50Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[100ecf2](https://github.com/OfficeDev/office-js-docs-pr/pull/886/commits/100ecf21ec93f97f74792afc87783c746dbcc93c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/register-sso-add-in-aad-v2.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/develop/register-sso-add-in-aad-v2.md) | :white_check_mark:Succeeded | |
[docs/develop/sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/develop/sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/includes/grant-admin-consent-to-an-add-in-include.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/includes/grant-admin-consent-to-an-add-in-include.md) | :white_check_mark:Succeeded | |
[docs/includes/register-sso-add-in-aad-v2-include.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/includes/register-sso-add-in-aad-v2-include.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C10%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904102039492257-886%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=nwCqsPraX3a%2FJ4Myjx3Z46JXW3pS%2FEFgMYOLN0midAA%3D&st=2019-04-10T20%3A36%3A10Z&se=2019-05-11T20%3A41%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[4cda58d](https://github.com/OfficeDev/office-js-docs-pr/pull/886/commits/4cda58d2c432ddcee51311c515957cee5b3d281c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/register-sso-add-in-aad-v2.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/develop/register-sso-add-in-aad-v2.md) | :white_check_mark:Succeeded | |
[docs/develop/sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/develop/sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/includes/grant-admin-consent-to-an-add-in-include.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/includes/grant-admin-consent-to-an-add-in-include.md) | :white_check_mark:Succeeded | |
[docs/includes/register-sso-add-in-aad-v2-include.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/includes/register-sso-add-in-aad-v2-include.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C10%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904102040524181-886%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=CXPFKDP6nwW%2BlasAxU7JdUMiHBORkeZMVWKPhm20d84%3D&st=2019-04-10T20%3A37%3A41Z&se=2019-05-11T20%3A42%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[291dab1](https://github.com/OfficeDev/office-js-docs-pr/pull/886/commits/291dab105474c6872489193e222ab351695d0028)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/register-sso-add-in-aad-v2.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/develop/register-sso-add-in-aad-v2.md) | :white_check_mark:Succeeded | |
[docs/develop/sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/develop/sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/includes/grant-admin-consent-to-an-add-in-include.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/includes/grant-admin-consent-to-an-add-in-include.md) | :white_check_mark:Succeeded | |
[docs/includes/register-sso-add-in-aad-v2-include.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/includes/register-sso-add-in-aad-v2-include.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C10%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904102045101690-886%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=1ZkXoJqzErh1xmzANKVKBu1DtrNS8exLYEebiOQGLeo%3D&st=2019-04-10T20%3A41%3A38Z&se=2019-05-11T20%3A46%3A38Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[21914ae](https://github.com/OfficeDev/office-js-docs-pr/pull/886/commits/21914aea75868c3ea94cad953e8d69afc8109798)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/register-sso-add-in-aad-v2.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/develop/register-sso-add-in-aad-v2.md) | :white_check_mark:Succeeded | |
[docs/develop/sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/develop/sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/includes/grant-admin-consent-to-an-add-in-include.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/includes/grant-admin-consent-to-an-add-in-include.md) | :white_check_mark:Succeeded | |
[docs/includes/register-sso-add-in-aad-v2-include.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/includes/register-sso-add-in-aad-v2-include.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C10%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904102051451244-886%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=hM%2FUpJuw%2Boarax3as1aV1MuHSSpkbU1DfH7YGWW4kpU%3D&st=2019-04-10T20%3A48%3A10Z&se=2019-05-11T20%3A53%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[a0856b8](https://github.com/OfficeDev/office-js-docs-pr/pull/886/commits/a0856b8f6cb6de3cb89212b72da61ca9c26ecd78)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/register-sso-add-in-aad-v2.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/develop/register-sso-add-in-aad-v2.md) | :white_check_mark:Succeeded | |
[docs/develop/sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/develop/sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/includes/grant-admin-consent-to-an-add-in-include.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/includes/grant-admin-consent-to-an-add-in-include.md) | :white_check_mark:Succeeded | |
[docs/includes/register-sso-add-in-aad-v2-include.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/includes/register-sso-add-in-aad-v2-include.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C10%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904102053590637-886%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=9cfLQPaXEiE72%2F6P3ZhCtCMkdpNhrY7gTqhQSN%2F0ie0%3D&st=2019-04-10T20%3A50%3A24Z&se=2019-05-11T20%3A55%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/includes/register-sso-add-in-aad-v2-include.md [hunk 1]
```diff
@@ -1,46 +1,43 @@
...
+    > If you get an error saying that the domain is already owned but you own it, follow the procedure at [Quickstart: Add a custom domain name to Azure Active Directory](/azure/active-directory/add-custom-domain) to register it, and then repeat this step. (This error can also occur if you are not signed in with credentials of an admin in the Office 365 tenancy. See step 2. Sign out and sign in again with admin credentials and repeat the process from step 3.)
+
+1. Select the **Add a scope** button. In the panel that opens, enter `access_as_user` as the **Scope name**. Set **Who can consent?** to the preferred option for your add-in then fill in the remaining fields with appropriate descriptions. Ensure that **State** is set to **Enabled**.
```
Comment (Rick-Kirkham) [inline docs/includes/register-sso-add-in-aad-v2-include.md]:
```suggestion
1. Select the **Add a scope** button. In the panel that opens, enter `access_as_user` as the **Scope name**. 

1. Set **Who can consent?** to **Admins and users**.

1. Fill in the fields for configuring the admin and user consent prompts with values that are appropriate for the `access_as_user` scope which enables the Office host application to use your add-in's web APIs with the same rights as the current user. Suggestions:

- **Admin consent title:** Office can act as the user.
- **Admin consent description:** Enable Office to call the add-in's web APIs with the same rights as the current user.
- **User consent title:** Office can act as you.
- **Admin consent description:** Enable Office to call the add-in's web APIs with the same rights as you have.

1. Ensure that **State** is set to **Enabled**.

1. Select **Add scope**.
```

File: docs/includes/register-sso-add-in-aad-v2-include.md [hunk 2]
```diff
@@ -1,46 +1,43 @@
...
+    > The domain part of the **Scope name** displayed just below the text field should automatically match the **Application ID URI** set in the previous step, with `/access_as_user` appended to the end; for example, `api://localhost:6789/c6c1f32b-5e55-4997-881a-753cc1d563b7/access_as_user`.
 
-1. In the **Pre-authorized applications** section, you identify the applications that you want to authorize to your add-in's web application. Each of the following IDs needs to be pre-authorized. Each time you enter one, a new empty textbox appears. (Enter only the GUID.)
+1. In the **Authorized client applications** section, you identify the applications that you want to authorize to your add-in's web application. Each of the following IDs needs to be pre-authorized. To authorize each one, select **Add a client application** button then, in the panel that opens, set the **Client ID** to the respective GUID and check the box for `api://$FQDN-WITHOUT-PROTOCOL$/$App ID GUID$/access_as_user`.
```
Comment (Rick-Kirkham) [inline docs/includes/register-sso-add-in-aad-v2-include.md]:
```suggestion
1. In the **Authorized client applications** section, you identify the applications that you want to authorize to your add-in's web application. Each of the following IDs needs to be pre-authorized. For each, take these steps:

  a. Select **Add a client application** button then, in the panel that opens, set the **Client ID** to the respective GUID and check the box for `api://$FQDN-WITHOUT-PROTOCOL$/$App ID GUID$/access_as_user`.
  b. Select **Add application**.
  
```

File: docs/includes/register-sso-add-in-aad-v2-include.md [hunk 3]
```diff
@@ -1,46 +1,43 @@
...
     * `d3590ed6-52b3-4102-aeff-aad2292ab01c` (Microsoft Office)
     * `57fb890c-0dab-4253-a5e0-7188c88b2bb4` (Office Online)
     * `bc59ab01-8403-45c6-8796-ac3ef710b3e3` (Office Online)
 
-1. Open the **Scope** drop-down beside each **Application ID** and check the box for `api://$FQDN-WITHOUT-PROTOCOL$/$App ID GUID$/access_as_user`.
+1. Select **Authentication** under **Manage**. In the **Redirect URIs** section, select **Web** in the **Type** dropdown then set the **Redirect URI** value to `https://$FQDN-WITHOUT-PROTOCOL$`.
```
Comment (Rick-Kirkham) [inline docs/includes/register-sso-add-in-aad-v2-include.md]:
```suggestion
1. Select **Authentication** under **Manage**. In the **Redirect URIs** section, select **Web** in the **Type** dropdown then set the **Redirect URI** value to `https://$FQDN-WITHOUT-PROTOCOL$`.

1. Select **Save** at the top of the form.
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[17ea081](https://github.com/OfficeDev/office-js-docs-pr/pull/886/commits/17ea0812c88c6e306b488d4ac5817bb672d48006)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/register-sso-add-in-aad-v2.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/develop/register-sso-add-in-aad-v2.md) | :white_check_mark:Succeeded | |
[docs/develop/sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/develop/sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/includes/grant-admin-consent-to-an-add-in-include.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/includes/grant-admin-consent-to-an-add-in-include.md) | :white_check_mark:Succeeded | |
[docs/includes/register-sso-add-in-aad-v2-include.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/includes/register-sso-add-in-aad-v2-include.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C15%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904151607230226-886%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=o37GGV97BTJec25Wxc1eUtSYu9qymmgZl7O0ZNX0jzk%3D&st=2019-04-15T16%3A03%3A56Z&se=2019-05-16T16%3A08%3A56Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[46487bc](https://github.com/OfficeDev/office-js-docs-pr/pull/886/commits/46487bcfccfe554b6154cd94e74c3f33dc8592e7)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/register-sso-add-in-aad-v2.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/develop/register-sso-add-in-aad-v2.md) | :white_check_mark:Succeeded | |
[docs/develop/sso-in-office-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/develop/sso-in-office-add-ins.md) | :white_check_mark:Succeeded | |
[docs/includes/grant-admin-consent-to-an-add-in-include.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/includes/grant-admin-consent-to-an-add-in-include.md) | :white_check_mark:Succeeded | |
[docs/includes/register-sso-add-in-aad-v2-include.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-update-aad-portal-info/docs/includes/register-sso-add-in-aad-v2-include.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C15%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904151625261339-886%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=yTatCKB3Db6BFySfMdYZkWzuz%2FAyrerZHlYb7%2FCmp0w%3D&st=2019-04-15T16%3A21%3A51Z&se=2019-05-16T16%3A26%3A51Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**